cloud/scaling.rst

```diff
@@ -1,3 +1,21 @@
+Provisioning
+############
+
+Once you've created your account at `https://console.citusdata.com <https://console.citusdata.com>`_ you can provision your Citus cluster. When you login you'll be at the home of the dashboard, and from here you can click New Formation to begin your formation creation.
+
+.. image:: ../images/cloud_provisioning.png
+
+Configuring Your Plan
+---------------------
+
+Citus Cloud plans vary based on the size of your primary node, size of your distributed nodes, number of distributed nodes and whether you have high availability or not. From within the Citus console you can configure your plan or you can preview what it might look like within the `pricing calculator <https://console.citusdata.com/pricing>`_.
+
+The key items you'll care about for each node:
+
+- Storage - All nodes come with 512 GB of storage
+- Memory - The memory on each node varies based on the size of node you select
+- Cores - The cores on each node varies based on the size of node you select
```

Comment:
Physical cpu cores? (We make a deliberate attempt to use physical rather than virtual cores. So I just wanted to ask.)